packages/manager/modules/netapp/src/onboarding/translations/Messages_fr_FR.json

```diff
@@ -14,5 +14,9 @@
   "netapp_onboarding_volumes_guides": "Guides",
   "netapp_onboarding_storage_title": "Stockage et sauvegarde",
   "netapp_onboarding_volume_acl_title": "Gestion des ACL",
-  "netapp_onboarding_volume_snapshots_title": "Gestion des Snapshots"
+  "netapp_onboarding_volume_snapshots_title": "Gestion des Snapshots",
+  "netapp_onboarding_bullet_point_1": "Un service de stockage fichiers entièrement managé par OVHcloud, hautes performances et hautement disponible, basé sur le système de fichiers NetApp ONTAP.",
+  "netapp_onboarding_bullet_point_2": "Capacité de stockage de 1 à 58 To par service et provisionnement des performances garanties au téraoctet. <strong>64 Mo/s et 4000 IOPS par To de capacité provisionnée.</strong>",
```

Comment:
**nitpick:** Since `64 Mo/s et 4000 IOPS par To de capacité provisionnée.` is a complete sentence. Better to add it as a new trans key and we can avoid `strong` tag in Trans files.